### PR TITLE
everything up to 3.0.0 (almost)

### PR DESCRIPTION
**Pull request by WILLATRONIX**
### **updated:**
    tinker list
    elevation
    clipboard
    gizmos
    painter
    noise painter
    all builder tools
    images
    + more

### **added:**
    display entities
    windows category
    palette
    tool options
    marker entities
    stamp
    unlocked rotation/scale
    lasso select
    block attributes
    + more

### **removed:** 
    clentaminator

### **fixed:**
    broken links
    beta version notice for text tool
    missing information for shape tool

### **things that still require updates**
    shape tool - some missing placement options
    path tool - hasnt been updated 
    shape tool - shape categories has very little info
    painter mode - further information on different modes
    commercial - managing and distributing licenses across a server
    multiplayer - incompatible plugins

### CODE REVIEW

**Comment by TheWarior73** [comment]
Hi, I have noticed some broken links in the current V of the docs, and found your PR, just to make sure that the links are working as intended, let's take an example :
When in `src/advanced/configuration.md` you link the `Blueprints` using the following link :
```md
The folders are used to store saved [Blueprints](/editor/blueprints.md), ...........
```
Will it link properly when built ? since you are starting from the `root/editor/...` there is a problem with viewing it from the GitHub.
Just wanted to make sure 👍  

Aline aka Th3_Warior

**Comment by WILLATRONIX** [comment]
> Hi, I have noticed some broken links in the current V of the docs, and found your PR, just to make sure that the links are working as intended, let's take an example : When in `src/advanced/configuration.md` you link the `Blueprints` using the following link :
> 
> ```
> The folders are used to store saved [Blueprints](/editor/blueprints.md), ...........
> ```
> 
> Will it link properly when built ? since you are starting from the `root/editor/...` there is a problem with viewing it from the GitHub. Just wanted to make sure 👍
> 
> Aline aka Th3_Warior

**Comment by WILLATRONIX** [comment]
@TheWarior73  yes, [the config links](https://github.com/Moulberry/AxiomDocumentation/pull/22/commits/0afa4d54c6241275bc9647f01a3bdd55e637cffe) and [all other links](https://github.com/Moulberry/AxiomDocumentation/pull/22/commits/24a9bda91e0d416d31966f38a618d2bea7cc5892) were fixed in this pr

**Comment by TheWarior73** [comment]
@WILLATRONIX I saw your PR commits don't worry, I am not the type to comment for nothing...
I was actually wondering if the build would get the links incorrectly or not, since on the repo you cannot navigate using them (dut to the link not using SRC.
I will assume that it won't be a problem 👍 

Thank you for the answer

File: src/advanced/commands.md [hunk 1]
```diff
@@ -26,6 +26,32 @@ Commands are essential for management and debugging.
...
 
-`//pos1` and `//pos2` are part of Axiom, but are intended to be used alongside WorldEdit.
\ No newline at end of file
+`//pos1` and `//pos2` are part of Axiom, but are intended to be used alongside WorldEdit.
+
+## Multiplayer Commands
+
+### /axiompaperdebug [permission]
+
+- Used to check whether the player has the provided [permission](/multiplayer/permissions.md) and returns true/false.
```
Comment (Moulberry) [inline src/advanced/commands.md]: This isn't what the /axiompaperdebug command does

File: src/builder/blockattributes.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Edit Block Attributes
+
+The Edit Block Attributes menu can be used to configure multiple settings that alter gameplay.
+
+| Setting                      | Description                                                                                |
+| ---------------------------- | ------------------------------------------------------------------------------------------ |
+| Show Collision Mesh          | Creates a red outline for every block with a transparent hitbox like leaves or fences.     |
```
Comment (Moulberry) [inline src/builder/blockattributes.md]: Barriers are the main reason you'd use this, might be good to mention

File: src/builder/blockattributes.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+| Show Structure Void Blocks   | Adds a texture[^note2] for structure void blocks.                                          |
+| Expand Hitboxes to Full Cube | Sets all hitboxes to a full block. Affects blocks like slabs or buttons.                   |
+| Make Fluid Hitboxes Solid    | Allows you to break individual blocks of water with your fist.                             |
+| Prevent Interactions         | Stops the player from opening containers or interacting with blocks while holding a block. |
+
+## Notes
+
+[^note1]: ![Light Block Texture](/img/light_block_15.png)
+
+[^note2]: ![Light Block Texture](/img/structure_void.png)
```
Comment (Moulberry) [inline src/builder/blockattributes.md]: Wrong name, should be structure void texture

File: src/editor/windows/blueprints.md [hunk 4]
```diff
@@ -2,18 +2,34 @@
...
-## Loading blueprints
+## Blueprint Browser
+
+Once you have blueprints, you can view them in the **Blueprint Browser**. You'll be able to see thumbnails for all of your blueprints, as well as perform searches or filter by tags. Clicking a blueprint will 'open' it load into your [clipboard](clipboard.md).
+
+> Tip: You can go back and refresh by clicking the appropriate button next to the seach box.
+
+## File Format
+
+Blueprints are stored as `.bp` files in the [Configuration](/advanced/configuration.md). The blueprint file itself is a GZ[^note1] and can be opened in programs such as 7zip to reveal the NBT file inside. The thumbnail, author and tags are also stored within the `.bp` file.
```
Comment (Moulberry) [inline src/editor/windows/blueprints.md]: This is not how blueprint files are stored

File: src/intro.md [hunk 5]
```diff
@@ -11,3 +15,17 @@ To access the [Builder Mode](builder/intro.md) menu, press Left Alt
 To open/close the [Editor Mode](editor/intro.md), press Right Shift
 
 These keybinds can be adjusted in Minecraft's controls menu (Esc > Options > Controls)
+
+## Latest Update
```
Comment (Moulberry) [inline src/intro.md]: Probably shouldn't have latest update here as it'd require constant maintenance to keep updated. People can look at the changelogs on modrinth/the website anyways if they want this information

File: src/multiplayer/commercial.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
+# Commercial Usage
```
Comment (Moulberry) [inline src/multiplayer/commercial.md]: Please remove this page, I think it'll just cause more confusion

File: src/multiplayer/intro.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+# Multiplayer
+
+With multiplayer, you can access multiple exclusive features such as the ability to teleport to other players using the [Editor UI](/editor/intro.md).
+
+Axiom is available for Paper and Fabric servers. To use Axiom on a Paper server, you'll need to have the plugin on the server. You can download the plugin from [here](https://modrinth.com/plugin/axiom-paper-plugin). To use Axiom on a Fabric server, you can use the base mod from [here](https://modrinth.com/mod/axiom) and place it in the plugins folder.
+
+If you're using Axiom for commercial purposes, you'll need to purchase a [Commercial License](commercial.md).
```
Comment (Moulberry) [inline src/multiplayer/intro.md]: Change this to link directly to commercial license page

File: src/advanced/commands.md [hunk 8]
```diff
@@ -26,6 +26,32 @@ Commands are essential for management and debugging.
 
 - Returns Axiom's status and information on how to resolve issues.
 
+## Axiom Gamerule Commands
+
+These are some extra commands for controlling the world and its properties. These are not available in [Multiplayer](/multiplayer/intro.md).
```
Comment (Moulberry) [inline src/advanced/commands.md]: Only available on Fabric, not Paper

**Review by Moulberry — CHANGES_REQUESTED** [review]

**Review by Moulberry — APPROVED** [review]

**Pull request merged**